test/suite_test.go

```diff
@@ -50,6 +60,17 @@ var _ = BeforeSuite(func() {
 	qemuImage, err := downloader.DownloadImage()
 	Expect(err).ShouldNot(HaveOccurred())
 
+	publicKey, err := createSSHKeys()
+	if err != nil {
+		log.Error(err)
```

Comment:
I think you can use `Expect(err).ShouldNot(HaveOccurred())` but no hard feelings in a BeforeSuite :)

Reply:
ah yes will fix that